### PR TITLE
add example js project

### PR DESCRIPTION
**Pull request by meskill**

### CODE REVIEW

File: .github/workflows/benchmark-pull-request.yaml [hunk 1]
```diff
@@ -22,6 +22,10 @@ jobs:
       - name: Install Stable Toolchain
         uses: actions-rust-lang/setup-rust-toolchain@v1
 
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
```
Comment (tusharmath) [inline .github/workflows/benchmark-pull-request.yaml]: No CI changes

File: projects/meskill/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (tusharmath) [inline projects/meskill/.gitignore]: Setup a gitignore in the root dir.

File: projects/meskill/src/index.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const { ApolloServer } = require('apollo-server');
+const typeDefs = require('./schema');
```
Comment (tusharmath) [inline projects/meskill/src/index.js:2]: Use `fs.readSync` to read the file from the root folder.

File: .github/workflows/benchmark-pull-request.yaml (hunk 1 above)
Comment (meskill) [reply .github/workflows/benchmark-pull-request.yaml]: removed

File: projects/meskill/.gitignore (hunk 2 above)
Comment (meskill) [reply projects/meskill/.gitignore]: removed

File: projects/meskill/src/index.js (hunk 3 above)
Comment (meskill) [reply projects/meskill/src/index.js:2]: done

**Pull request merged**